ROS/Chapter_1.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<p>I still feel uneasy with this warning, haha</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Ill figure out how to get rid of it</p>

Reply:
not a big deal is more my problem really, haha.